### DISCUSSION TITLE
Ferramentas CSS (Além de Pré processadores)

### DISCUSSION THREAD

**Discussion by raphaguasta** [frontendbr/forum#350] [Discussão]
Pessoal!

Imagino que a grande maioria utilize sass/less/stylus. Mas além disso, estava lendo sobre duas ferramentas para melhorar o fluxo de desenvolvimento css, que são as seguintes:

- Rucksack [https://simplaio.github.io/rucksack/](https://simplaio.github.io/rucksack/)
- CSSNext [http://cssnext.io/](http://cssnext.io/)

Confesso que em ambos casos, só algumas features me empolgaram, mas não o suficiente pra implementar num projeto.

Vcs usam essas ferramentas? Valem a pena? Estão utilizando outras? Ou nada disso é realmente necessário?

O que acham?

Valeu! :)

**Comment by afonsopacifer** [comment]
Eu sou estranho.. fiz a minha própria tool rsrsr
https://github.com/bananacss/bananacss

fiz um plugin para gulp uns generator em slush e tentei ser feliz..
Mas não recomendo que faça isso não.. até pq hoje em dia eu não uso nada rsrsr uso CSS puro mesmo, devo ter falado sobre isso em alguma issue, vou achar para linkar aqu :)

**Comment by raphaguasta** [comment]
Hahaha legal a iniciativa @afonsopacifer!

CSS puro sem pré processador nenhum? Old school mesmo! rs

Achei uma outra ferramenta que me agradou muito mais que as iniciais:
- Short: [ https://github.com/jonathantneal/postcss-short](https://github.com/jonathantneal/postcss-short)

**Comment by suissa** [comment]
[Iniciamos a série sobe PostCSS](https://www.youtube.com/playlist?list=PL77JVjKTJT2jLGUXW5TVKDbSALbm1ovUH)

**Comment by felipefialho** [comment]
Olha só @raphaguasta 
- [Apresento o Rucksack](http://www.felipefialho.com/blog/2015/apresento-o-rucksack)

Escrevi ano passado, continuo usando e é sensacional.

**Comment by brunoalmeida-dev** [comment]
@LFeh o font-size responsivo é a melhor coisa do Rucksack <3

**Comment by felipefialho** [comment]
@brunob182 Uso para ajudar a fazer transitions, para auto-prefix, para facilitar fontes responsivas, para agrupar positions.... uso bastante, hehe.

**Comment by afonsopacifer** [comment]
@raphaguasta Entao mano.. na verdade não é tão old school rsrs, pq eu uso basicamente todas as features dos preprocessors.. mas isso é assunto para outra issue rsrsr

**Comment by raphaguasta** [comment]
Legal @LFeh  @brunob182! Acho que vou fazer um teste com ele! :)

Achei esse aqui tb, tem algumas coisas bem legais (apesar de não ter font responsiva =( ): https://github.com/jonathantneal/postcss-short

Gostei de um detalhe específico em uma das features dele, ao usar a feature de "shorthand" podemos usar um \* ...
Exemplo: `position: fixed 10em 5em * 2em`
Assim não preciso obrigatoriamente setar um **0**, o \* 'pula' esse lado, bem legal! No caso do Rucksack não achei se isso tbm é possível, sabem me dizer?

@afonsopacifer Hahahah. Opa! Abre lá a gente conversa 😄

**Comment by felipefialho** [comment]
@raphaguasta O Rucksack tambem tem isso :p 

```
position absolute 0
```
